### ISSUE TITLE
Continuous popups about connection to remote being down

### ISSUE THREAD

**Issue by vis256** [GitFiend/Support#199]
When our server hosting git repos went down **every** time I alt-tabbed to GitFiend it showed a popup with error message. It's not a big deal but it would be cool if connection errors were displayed somewhere in main window without having to close the popup every time.

**Comment by GitFiend** [comment]
Yeah, that sounds annoying. Thanks for reporting. I need to handle this case, as well as when there's no network connection. In the meantime, you can disable auto fetch by right clicking on fetch.
